src/main/java/org/braekpo1nt/mctmanager/games/mecha/MechaGame.java

```diff
@@ -335,7 +342,25 @@ public void onPlayerDeath(PlayerDeathEvent event) {
             onTeamWin(winningTeam);
         }
     }
-    
+
+    @EventHandler
+    public void onPlayerInteractEntity(PlayerInteractEntityEvent event) {
+        if (!gameActive) {
+            return;
+        }
+        if (!mechaHasStarted) {
```

Comment:
Funny little change here, but I want you to remove this check. It should only check if the gameActive boolean is true. 

This is because in future situations, the map-makers might change the map such that players can reach armor stands and picture frames from their starting positions, during the countdown, (before `mechaHasStarted==true`).